rpp30-boc-meltdown/biteshare
# Session menu

## Description

Create Session UI

## How to test

When click on the session menu, the list of the menu will show up.
user can select the item by click (press) on the item and check mark will show on the left column
When the user click 'I'm ready' it will take the user to summary screen
## Notes


## Issue tracking

https://trello.com/c/COVnqgzK/39-ui-implementation-of-session-menu-using-mock-data
